system/Database/Forge.php

```diff
@@ -374,15 +443,12 @@ public function createTable($table, $if_not_exists = false, array $attributes =
 				{
 					throw new DatabaseException('This feature is not available for the database you are using.');
 				}
-
 				return false;
 			}
 		}
-
 		if (($result = $this->db->query($sql)) !== false)
 		{
 			empty($this->db->dataCache['table_names']) OR $this->db->dataCache['table_names'][] = $table;
-
```

Comment:
Why are you deleting all blank lines? space between elements is good for readability.